composer.json

```diff
@@ -19,8 +19,8 @@
         "guzzlehttp/guzzle": "^5.3 || ^6 || ^7",
         "guzzlehttp/promises": "~1.0",
         "guzzlehttp/psr7": "^1.4.1",
-        "illuminate/notifications": "^6.0 || ^7.0 || ^8.0",
-        "illuminate/support": "^6.0 || ^7.0 || ^8.0",
+        "illuminate/notifications": "^6.0 || ^7.0 || ^8.0 || ^9.0",
+        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0",
```

Comment:
you need to also check the other main dependencies like guzzle, uuid, phpunit etc. they all change their version requirements with each Laravel version. phpunit 8.0 is not supported since L8.

Reply:
@datashaman updated phpunit. Guzzle and uuid seem ok